app/js/components/SearchBar.js

```diff
@@ -40,6 +40,7 @@ export default class SearchBar extends Component {
           ref={input => { this.input = input; }}
           onChange={onInputChange}
           value={searchInput}
+          autofocus=autofocus
```

Comment:
I'm pretty sure this has to be either:

`autofocus={true}`

Oor just

`autofocus`

with no value.